content/docs/project/file/contents.lr

```diff
@@ -89,6 +89,11 @@ This section controls some basics about the project:
 > Note that if this setting is used some functionality in the desktop app
 > might no longer work (for instance opening `.lr` files with a double click).
 
+`output_path`
+> Configure the build path for your project. By default, Lektor will detect a
+> proper location depeding on your operation system, only use this setting if
+> you want to set a custom path.
```

Comment:
Maybe add something about how relative paths work? And that absolute paths are allowed but not portable.